packages/integration-testsuite/src/httpServerTests.ts

```diff
@@ -382,7 +382,7 @@ export function defineIntegrationTestSuiteHttpServerTests(
         return req.then((res) => {
           expect(res.status).toEqual(400);
           expect(
-            ['Unexpected token f', 'Bad Request', 'Invalid JSON'].some(
+            ['in JSON at position 1', 'Bad Request', 'Invalid JSON'].some(
```

Comment:
Error message changed in Node 20, this substring is compatible across all of our supported versions.